XIPs/xip-53-v2-deprecation.md

```diff
@@ -0,0 +1,76 @@
+---
+xip: 53
+title: XMTP V2 deprecation
+description: This XIP defines a plan to turn off writes to the XMTP V2 network and making it read only to encourage developers to upgrade to using the XMTP V3 network.
+author: Naomi Plasterer (@nplasterer)
+discussions-to: TBD
+status: Draft
+type: Standards
+category: XRC
+created: 2025-01-29
+---
+
+## Abstract
+
```

Comment:
Took feedback @cameronvoell in the Notion and rewrote a bit - WDYT?

Reply:
looks great 🙏